crates/libs/core/src/types/common/security_credentials/mod.rs

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Where are denies needed?

Reply:
I don't see unsafe code in this mod in this PR. Assme it will be used in the next.